src/environmentbase/scripts/instance_type_scraper.py

```diff
@@ -45,7 +45,7 @@ def get_instance_types(tbl):
 
 def build_type_to_arch_map(tbl, arch_type, map):
     for it in get_instance_types(tbl):
-        map[it] = {'Arch': arch_type}
+        map[it] = arch_type
```

Comment:
The effort is just `git checkout master -- src/environmentbase/scripts/instance_type_scraper.py`